db/migrate/20160402015808_alter_sections_add_description_column.rb

```diff
@@ -1,5 +1,5 @@
 class AlterSectionsAddDescriptionColumn < ActiveRecord::Migration
   def change
-    add_column :sections, :description, :string
+    add_column :sections, :description, :string, null: false
```

Comment:
We shouldn't modify existing migrations.  Only add new ones to modify the database.